src/torchcodec/decoders/_core/fetch_and_expose_non_gpl_ffmpeg_libs.cmake

```diff
@@ -8,33 +8,150 @@ if (CMAKE_VERSION VERSION_GREATER_EQUAL "3.24.0")
 endif()
 
 include(FetchContent)
+
 set(
     base_url
-    https://pytorch.s3.amazonaws.com/torchcodec/ffmpeg/2024-06-11/linux_x86_64
+    https://pytorch.s3.amazonaws.com/torchcodec/ffmpeg/2024-09-13
 )
+
+if (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+    set(
+        platform_url
+        ${base_url}/linux_x86_64
+    )
+
+    set(
+        f4_hash
```

Comment:
nit: maybe use a more descriptive name here?

Reply:
Sure, let's call it fn_sha256.